docs/user-guide/02_physics/01_plasma.md

```diff
@@ -75,42 +80,46 @@ and where the `fields` group has the following datasets
 | `er`                | Float64 | 2    | [`nr`,`nz`] | V/m   | Radial component of the electric field                        |
 | `et`                | Float64 | 2    | [`nr`,`nz`] | V/m   | Torodial/Phi component of the electric field                  |
 | `ez`                | Float64 | 2    | [`nr`,`nz`] | V/m   | Z component of the electric field                             |
+| `description`       | String  | 0    | NA          | NA    | Electromagnetic Field                    |
+| `coordinate system` | String  | 0    | NA          | NA    | Cylindrical                              |
 
 # Distributions
 The fast-ion distribution function typically does not have a functional form.
 This means that we need to input the fast-ion distribution function into FIDASIM.
 FIDASIM accepts three different types of fast-ion distributions.
 
-1. Guiding Center Distribution Function: F(E,p,R,Z)
+1. Guiding Center Distribution Function: F(E,p,R,Z,Phi)
```

Comment:
To indicate that `Phi` is optional use the following notation.
`1. Guiding Center Distribution Function: F(E,p,R,Z[,Phi])`